fluent-plugin-google-cloud.gemspec

```diff
@@ -23,7 +23,9 @@ eos
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
   gem.add_runtime_dependency 'google-api-client', '~> 0.9.0'
   gem.add_runtime_dependency 'google-cloud-logging', '~> 0.23.2'
-  gem.add_runtime_dependency 'googleauth', '~> 0.4'
+  # Pin the gem at earlier versions because this bug was introduced in v0.5.2:
```

Comment:
If you have to comment at all, I'd say:
```ruby
# Avoid v0.5.2 because of a bug: https://github.com/google/google-auth-library-ruby/issues/113
```
But you may not even need that, because `git blame` clearly shows the commit that added this constraint, which will refer to this PR (#140), which will then refer to the upstream bug (https://github.com/google/google-auth-library-ruby/issues/113) in the description.